<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CalendarListApp</class>
 <widget class="QMainWindow" name="CalendarListApp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>605</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icons.qrc">
    <normaloff>:/icons/vectors/calendarVector.png</normaloff>:/icons/vectors/calendarVector.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255 );</string>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(49, 162, 125);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="lineWidth">
       <number>0</number>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="calendarPage">
       <widget class="QCalendarWidget" name="calendarWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>0</y>
          <width>461</width>
          <height>221</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Calibri Light</family>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <italic>false</italic>
          <bold>true</bold>
          <kerning>true</kerning>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(105, 98, 140);</string>
        </property>
        <property name="minimumDate">
         <date>
          <year>2000</year>
          <month>9</month>
          <day>14</day>
         </date>
        </property>
        <property name="maximumDate">
         <date>
          <year>2200</year>
          <month>12</month>
          <day>31</day>
         </date>
        </property>
        <property name="gridVisible">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QTableWidget" name="tableWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>270</y>
          <width>461</width>
          <height>271</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
        <column>
         <property name="text">
          <string>Date</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>To Do's   </string>
         </property>
        </column>
       </widget>
       <widget class="QPushButton" name="clearBtn1">
        <property name="geometry">
         <rect>
          <x>380</x>
          <y>550</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 75 10pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(68, 187, 193);</string>
        </property>
        <property name="text">
         <string>Clear</string>
        </property>
       </widget>
       <widget class="QPushButton" name="saveBtn1">
        <property name="geometry">
         <rect>
          <x>260</x>
          <y>550</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;After every item you enter press here&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(255, 166, 224);</string>
        </property>
        <property name="text">
         <string>Save</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>230</y>
          <width>461</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="workPage">
       <widget class="QLineEdit" name="inputBox">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>60</y>
          <width>461</width>
          <height>41</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
        <property name="clearButtonEnabled">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="addBtn">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>120</y>
          <width>111</width>
          <height>31</height>
         </rect>
        </property>
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Press to add to the list&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(194, 165, 255);</string>
        </property>
        <property name="text">
         <string>Add</string>
        </property>
       </widget>
       <widget class="QPushButton" name="deleteBtn">
        <property name="geometry">
         <rect>
          <x>240</x>
          <y>120</y>
          <width>111</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(157, 132, 191);</string>
        </property>
        <property name="text">
         <string>Delete</string>
        </property>
       </widget>
       <widget class="QListWidget" name="listWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>170</y>
          <width>461</width>
          <height>391</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">selection-background-color: rgb(255, 218, 205);
background-color: rgb(255, 255, 255);
font: 9pt &quot;MS Shell Dlg 2&quot;;</string>
        </property>
       </widget>
       <widget class="QPushButton" name="clearBtn">
        <property name="geometry">
         <rect>
          <x>360</x>
          <y>120</y>
          <width>111</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 9pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(185, 156, 226);</string>
        </property>
        <property name="text">
         <string>Clear List</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>181</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>To Do List</string>
        </property>
       </widget>
       <widget class="QPushButton" name="saveBtn">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>120</y>
          <width>101</width>
          <height>31</height>
         </rect>
        </property>
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Press to save the list&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(209, 176, 255);
</string>
        </property>
        <property name="text">
         <string>Save</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="bookPage">
       <widget class="QPushButton" name="addBtn_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>120</y>
          <width>111</width>
          <height>31</height>
         </rect>
        </property>
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Press to add to the list&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(255, 178, 179);</string>
        </property>
        <property name="text">
         <string>Add</string>
        </property>
       </widget>
       <widget class="QPushButton" name="deleteBtn_2">
        <property name="geometry">
         <rect>
          <x>240</x>
          <y>120</y>
          <width>111</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(223, 141, 166);</string>
        </property>
        <property name="text">
         <string>Delete</string>
        </property>
       </widget>
       <widget class="QPushButton" name="clearBtn_2">
        <property name="geometry">
         <rect>
          <x>360</x>
          <y>120</y>
          <width>111</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 9pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(200, 109, 112);</string>
        </property>
        <property name="text">
         <string>Clear List</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="inputBox_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>60</y>
          <width>461</width>
          <height>41</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
        <property name="clearButtonEnabled">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QListWidget" name="listWidget_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>170</y>
          <width>461</width>
          <height>391</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>181</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Book List</string>
        </property>
       </widget>
       <widget class="QPushButton" name="saveBtn_2">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>120</y>
          <width>101</width>
          <height>31</height>
         </rect>
        </property>
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Press to save the list&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(255, 139, 141);</string>
        </property>
        <property name="text">
         <string>Save</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="shoppingPage">
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>181</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Shopping List</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="inputBox_3">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>60</y>
          <width>461</width>
          <height>41</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
        <property name="clearButtonEnabled">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="addBtn_3">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>120</y>
          <width>111</width>
          <height>31</height>
         </rect>
        </property>
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Press to add to the list&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(192, 255, 219);</string>
        </property>
        <property name="text">
         <string>Add</string>
        </property>
       </widget>
       <widget class="QPushButton" name="deleteBtn_3">
        <property name="geometry">
         <rect>
          <x>240</x>
          <y>120</y>
          <width>111</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(146, 255, 195);</string>
        </property>
        <property name="text">
         <string>Delete</string>
        </property>
       </widget>
       <widget class="QPushButton" name="clearBtn_3">
        <property name="geometry">
         <rect>
          <x>360</x>
          <y>120</y>
          <width>111</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 9pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(78, 204, 166);</string>
        </property>
        <property name="text">
         <string>Clear List</string>
        </property>
       </widget>
       <widget class="QListWidget" name="listWidget_3">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>170</y>
          <width>461</width>
          <height>391</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
       </widget>
       <widget class="QPushButton" name="saveBtn_3">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>120</y>
          <width>101</width>
          <height>31</height>
         </rect>
        </property>
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Press to save the list&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(74, 200, 173);</string>
        </property>
        <property name="text">
         <string>Save</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="travelPage">
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>181</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Travel List</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="inputBox_4">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>60</y>
          <width>461</width>
          <height>41</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
        <property name="clearButtonEnabled">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="addBtn_4">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>120</y>
          <width>111</width>
          <height>31</height>
         </rect>
        </property>
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Press to add to the list&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(181, 233, 255);
font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
        </property>
        <property name="text">
         <string>Add</string>
        </property>
       </widget>
       <widget class="QPushButton" name="clearBtn_4">
        <property name="geometry">
         <rect>
          <x>360</x>
          <y>120</y>
          <width>111</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 9pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(64, 168, 206);</string>
        </property>
        <property name="text">
         <string>Clear List</string>
        </property>
       </widget>
       <widget class="QPushButton" name="deleteBtn_4">
        <property name="geometry">
         <rect>
          <x>240</x>
          <y>120</y>
          <width>111</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(81, 185, 206);</string>
        </property>
        <property name="text">
         <string>Delete</string>
        </property>
       </widget>
       <widget class="QListWidget" name="listWidget_4">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>170</y>
          <width>461</width>
          <height>391</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
       </widget>
       <widget class="QPushButton" name="saveBtn_4">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>120</y>
          <width>101</width>
          <height>31</height>
         </rect>
        </property>
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Press to save the list&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(106, 196, 255);
font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
        </property>
        <property name="text">
         <string>Save</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="iconSize">
    <size>
     <width>75</width>
     <height>75</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_calendar"/>
   <addaction name="separator"/>
   <addaction name="actionWork_List"/>
   <addaction name="actionBook_List"/>
   <addaction name="actionShopping_List"/>
   <addaction name="actionTravel_List"/>
  </widget>
  <action name="action_calendar">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/vectors/calendarVector.png</normaloff>:/icons/vectors/calendarVector.png</iconset>
   </property>
   <property name="text">
    <string>Calendar</string>
   </property>
   <property name="toolTip">
    <string>Enter Calendar</string>
   </property>
   <property name="shortcut">
    <string>Shift+C</string>
   </property>
  </action>
  <action name="actionWork_List">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/vectors/suitcase.png</normaloff>:/icons/vectors/suitcase.png</iconset>
   </property>
   <property name="text">
    <string>To Do List</string>
   </property>
   <property name="toolTip">
    <string>Add your &quot;to-do's&quot; !</string>
   </property>
   <property name="shortcut">
    <string>Shift+W</string>
   </property>
  </action>
  <action name="actionBook_List">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/vectors/bookIcon.png</normaloff>:/icons/vectors/bookIcon.png</iconset>
   </property>
   <property name="text">
    <string>Book List</string>
   </property>
   <property name="toolTip">
    <string>Add your will-read books here.</string>
   </property>
   <property name="shortcut">
    <string>Shift+B</string>
   </property>
  </action>
  <action name="actionShopping_List">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/vectors/shoppingBag.png</normaloff>:/icons/vectors/shoppingBag.png</iconset>
   </property>
   <property name="text">
    <string>Shopping List</string>
   </property>
   <property name="toolTip">
    <string>Enter items you need to buy!</string>
   </property>
   <property name="shortcut">
    <string>Shift+S</string>
   </property>
  </action>
  <action name="actionTravel_List">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/vectors/travel.png</normaloff>:/icons/vectors/travel.png</iconset>
   </property>
   <property name="text">
    <string>Travel List</string>
   </property>
   <property name="toolTip">
    <string>Enter items you need on your travels.</string>
   </property>
   <property name="shortcut">
    <string>Shift+T</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
